opentracing-api/src/main/java/opentracing/propagation/Extractor.java

```diff
@@ -17,7 +17,7 @@
 
 /**
  */
-public interface Extractor<T> {
+    public interface Extractor<T> {
```

Comment:
sorry for the ultra-nitpick, but is there some Java whitespace convention happening here or is it just an errant `<Tab>`?